server/src/test/java/org/elasticsearch/search/query/ThrowingQueryBuilder.java

```diff
@@ -122,9 +120,21 @@ public int hashCode() {
             public void visit(QueryVisitor visitor) {
                 visitor.visitLeaf(this);
             }
+
+            @Override
+            public Query rewrite(IndexSearcher indexSearcher) throws IOException {
+                maybeThrow(context);
```

Comment:
Had to adjust this since some tests without data started failing on account that we weren't creating the weight any longer.